environments/production/tofu/main.tf

```diff
@@ -7,6 +7,7 @@ module "cluster" {
     source = "../../site/tofu/"
 
     cluster_name = "slurm-production"
+    cluster_image_id = "8cdcd255-c6c2-4131-9901-6a6b25b859f0" # openhpc-250910-1710-f605b7d8
```

Comment:
As of this PR, there's a different image for staging and production. It feels like it's best to update the image in `site` so that we know it's always going to end up in sync between the two of them.

Reply:
I've pushed a commit to do this